tests/end_to_end/test_ip_reputation/data/playbooks/configuration.yaml

```diff
@@ -0,0 +1,112 @@
+- name: Test manager configuration
+  hosts: wazuh-manager
+  become: true
+  tasks:
+
+    - name: Install apache
+      become: true
+      shell: yum install httpd -y
+
+    - name: Start apache
+      become: true
+      shell: systemctl start httpd
+
+    - name: Install pyhton
+      become: true
+      shell: yum install python39 -y
```

Comment:
Not all operating system versions will have a repository where python 3.9 can be installed.

This can cause problems depending on the image being used.

Reply:
Done in 0b80f86